### DISCUSSION TITLE
Improving the documentation of the library

### DISCUSSION THREAD

**Discussion by GillesTourreau** [Azure/communication-ui-library#3948] [General]
Hello the team!

I would like to participate to improve the documentation of the library.
- Is it possible to participate with a fork repository and submitting a PR ?
- Can you confirm that the source code of the demo website of the [communication-ui-library](https://azure.github.io/communication-ui-library/?path=/story/overview--page) library is located in the [/packages/storybook](https://github.com/Azure/communication-ui-library/tree/main/packages/storybook) folder?
- Does the API documentation on the Microsoft [@azure/communication-reac](https://learn.microsoft.com/en-us/javascript/api/@azure/communication-react/) web site is retrieved from the comments JSDoc of the source code ?
   - Is it updated when a new version is published on npm.org ?
   - Why some `@link` are broken when publishing on the Microsoft website ?
![image](https://github.com/Azure/communication-ui-library/assets/29240370/ad7893c3-a0b8-4d38-8256-676d6b0aad2b)

**Comment by alkwa-msft** [comment]
We would be absolutely happy to have some contributions. It is challenging for us to build out everything together and we are happy to help review any pull requests others may have. There may be some issues preventing us from taking on pull requests normally but yes if you want to go through the process of forking and creating a pull request to start we can start to work with you when more of us are back in January.

**Is it possible to participate with a fork repository and submitting a PR ?**
Yes
**Can you confirm that the source code of the demo website of the [communication-ui-library](https://azure.github.io/communication-ui-library/?path=/story/overview--page) library is located in the [/packages/storybook](https://github.com/Azure/communication-ui-library/tree/main/packages/storybook) folder?**
The storybook isn't really considered a demo "site" as much as it is documentation. You are correct in the location of the source code. The source code for this storybook is located in packages/storybook
**Does the API documentation on the Microsoft [@azure/communication-reac](https://learn.microsoft.com/en-us/javascript/api/@azure/communication-react/) web site is retrieved from the comments JSDoc of the source code ?**
I will need to get back to you on this question
**Is it updated when a new version is published on npm.org ?**
Right now we are only updating our documentation once a month which co-insides with our beta release. So every beta release to npm. We can look into more updating of documentation as we consider additional items to improve our documentation in 2024.
**Why some @link are broken when publishing on the Microsoft website ?**
There is an issue with how documentation is being generated internally. We can look into seeing if this can be fixed up in the coming new year
